### PR TITLE
Added python solution for usaco-1210

### PR DESCRIPTION
**Pull request by Keys9873**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [_] I have tested my code.
- [_] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [_] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-1210.mdx [hunk 1]
```diff
@@ -146,5 +146,71 @@ int main() {
...
-
+	
+<PySection>
+```python
+import math
+
+# Function that skips forward x submission
+# with constants a and b and current expected value e
+# based on the geometric series formula
+def skip_submissions(a, b, e, x):
```
Comment (ryanchou-dev) [inline solutions/usaco-1210.mdx:157]: multiline comments?

Comment (SansPapyrus683) [reply solutions/usaco-1210.mdx:157]: not addressed

Comment (Keys9873) [reply solutions/usaco-1210.mdx]: done

File: solutions/usaco-1210.mdx [hunk 2]
```diff
@@ -146,5 +146,77 @@ int main() {
 ```
 
 </CPPSection>
-
+	
+<PySection>
+```python
+import math
+
+'''
```
Comment (SansPapyrus683) [inline solutions/usaco-1210.mdx:154]: double quotes

File: solutions/usaco-1210.mdx [hunk 3]
```diff
@@ -146,5 +146,77 @@ int main() {
...
+Function that skips forward x submission
+with constants a and b and current expected value e
+based on the geometric series formula
+'''
+def skip_submissions(a, b, e, x):
```
Comment (SansPapyrus683) [inline solutions/usaco-1210.mdx:159]: type annotations

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-1210.mdx (hunk 3 above)
Comment (Keys9873) [reply solutions/usaco-1210.mdx:159]: wdym?

File: solutions/usaco-1210.mdx (hunk 2 above)
Comment (Keys9873) [reply solutions/usaco-1210.mdx]: done

File: solutions/usaco-1210.mdx (hunk 3 above)
Comment (SansPapyrus683) [reply solutions/usaco-1210.mdx:159]: https://realpython.com/lessons/annotations/

Comment (Keys9873) [reply solutions/usaco-1210.mdx]: done

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/usaco-1210.mdx [hunk 4]
```diff
@@ -146,5 +146,79 @@ int main() {
...
+def skip_submissions(a: float, b: float, e: float, x: float) -> float:
+	"""
+	Function that skips forward x submission
+	with constants a and b and current expected value e
+	based on the geometric series formula
+	"""
+	return pow(a, x) * e + b * (pow(a, x) - 1) / (a - 1)
+
+
+t, k = map(int,input().split())
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1210.mdx:164]:
```suggestion
t, k = map(int, input().split())
```

File: solutions/usaco-1210.mdx [hunk 5]
```diff
@@ -146,5 +146,79 @@ int main() {
...
+	"""
+	cross = math.floor(e)
+
+	# Find a and b using the precomputed prefix sums
+	a = pref_prob[cross]
+	b = pref_exp[t] - pref_exp[cross]
+
+	# Binary search to see how many submissions we can skip forward
+	lo = 1
+	hi = 1e9
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1210.mdx:203]:
```suggestion
	hi = 10**9
```

File: solutions/usaco-1210.mdx [hunk 6]
```diff
@@ -146,5 +146,79 @@ int main() {
...
+
+# Subtract sample
+t -= 1
+
+# Precompute probablility of getting i test cases using pascal's identity 
+prob = [[0 for i in range(t + 1)] for j in range(t + 1)]
+prob[0][0] = 1
+for i in range(1, t + 1):
+	prob[i][0] = prob[i - 1][0] / 2
+	for j in range(1, t + 1):
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-1210.mdx:174]:
```suggestion

	for j in range(1, t + 1):
```

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]

File: solutions/usaco-1210.mdx [hunk 7]
```diff
@@ -201,7 +201,7 @@ while k != 0:
 
 	# Binary search to see how many submissions we can skip forward
 	lo = 1
-	hi = 1e9
+	hi = 10**9
```
Comment (SansPapyrus683) [inline solutions/usaco-1210.mdx:204]:
```suggestion
	hi = 10 ** 9
```

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
beep beep boop

**Pull request merged**